app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/InsufficientConfidenceLevelController.scala

```diff
@@ -47,18 +46,19 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class InsufficientConfidenceLevelController @Inject()(
-  val authenticatedAction: AuthenticatedAction,
-  val sessionDataAction: SessionDataAction,
-  val sessionStore: SessionStore,
-  val errorHandler: ErrorHandler,
-  val config: Configuration,
-  bprNameMatchService: BusinessPartnerRecordNameMatchRetryService,
-  sautrNameMatchRetryStore: BusinessPartnerRecordNameMatchRetryStore,
-  doYouHaveANinoPage: views.html.do_you_have_a_nino,
-  doYouHaveAnSaUtrPage: views.html.do_you_have_an_sa_utr,
-  enterSautrAndNamePage: views.html.enter_sa_utr_and_name,
-  tooManyUnsuccessfulNameMatchesPage: views.html.too_many_name_match_attempts,
-  cc: MessagesControllerComponents
+                                                       val authenticatedAction: AuthenticatedAction,
+                                                       val sessionDataAction: SessionDataAction,
+                                                       val sessionStore: SessionStore,
+                                                       val errorHandler: ErrorHandler,
+                                                       val config: Configuration,
+                                                       val auditService: AuditService,
```

Comment:
looks like you're not using the service in this controller

Reply:
no, as the auditing has been removed to the service layer as discussed previously